rod_cutting_problem/rodCuttingProblem.js

```diff
@@ -0,0 +1,33 @@
+/*
+* Problem Statement: Given a rod of length n and an array of prices that contains prices of all pieces of size smaller than n.
+* Determine the maximum value obtainable by cutting up the rod and selling the pieces.
+*/
+
+function rodCuttingProblem (price) {
+	/*
+	Computes maximum money that can be earned by cutting a rod of length n (Bottom-Up Approach).
+	Time Complexity	:	O(n ^ 2)
+	Space Complexity : O(n)
+	:param price:	Array in which price[i] denotes price	of rod of length i.
+	:return: returns maximum value obtainable by cutting up the rod and selling the pieces.
+	*/
+	let n = price.length;
+	let bestPrice = new Array(n + 1);
+	bestPrice.fill(0);
+	for (let i = 1; i < bestPrice.length; i++) {
+		bestPrice[i] = price[i - 1];
+	}
+	for (let i = 1; i <= n; i++) {
+		for (let j = i; j <= n; j++) {
```

Comment:
The inner for loop content is different so it gives correct ans.
Take a look @aviaryan 
This is similar to the java implementation that we have.